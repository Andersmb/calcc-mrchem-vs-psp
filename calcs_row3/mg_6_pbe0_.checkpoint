/cluster/projects/nn4654k/ambr/MWCheckpoints/5688400
