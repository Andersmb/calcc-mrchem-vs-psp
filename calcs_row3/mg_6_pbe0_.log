Starting job 5688400 on c3-6,c5-[33,40],c10-27,c11-25 at Fri May 20 11:25:54 CEST 2022

Copying mg_6_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/doublets
Copying mg_6_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/doublets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688400      mg_6_pbe0_        180                                             00:25:01      0:0 
5688400.bat+      batch         36        1   00:00:02          0   00:00:02   00:25:01      0:0 
5688400.ext+     extern        180        5   00:00:00          0   00:00:00   00:25:01      0:0 
5688400.0      mrchem.x        180       30   00:24:49         29   00:49:06   00:24:53      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688400                                                                          
5688400.bat+     19280K          0     19280K        0              0          0 
5688400.ext+          0          0          0        0              0          0 
5688400.0      5287952K         29 2866023082        0              4          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688400                                                                                               
5688400.bat+       19.41M               0         19.41M        0.06M                0          0.06M 
5688400.ext+        0.00M               0          0.00M            0                0              0 
5688400.0           7.32M              24          6.96M       63.09M                9         29.77M 

Job 5688400 completed at Fri May 20 11:50:56 CEST 2022
