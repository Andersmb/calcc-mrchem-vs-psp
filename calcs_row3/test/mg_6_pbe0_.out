                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Li            :        0.000000        0.000000       -4.562551
    1      Mg            :        0.000000        0.000000        1.302969
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               8
 Beta electrons          :                                               7
 Total electrons         :                                              15
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999993831e-01
    1    1    a          :                              9.999999977519e-01
    2    1    a          :                              9.999993945394e-01
    3    1    a          :                              1.000000015032e+00
    4    1    a          :                              9.999999999023e-01
    5    1    a          :                              9.999999999023e-01
    6    1    a          :                              1.000000000491e+00
    7    1    a          :                              1.000000013603e+00
    8    1    b          :                              9.999999994846e-01
    9    1    b          :                              9.999999952565e-01
   10    1    b          :                              9.999998750605e-01
   11    1    b          :                              9.999999985476e-01
   12    1    b          :                              9.999999999023e-01
   13    1    b          :                              9.999999999023e-01
   14    1    b          :                              1.000000000611e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                205.133813442277
 E-N energy              :            (au)               -505.343857195469
 Coulomb energy          :            (au)                105.977974227948
 Exchange energy         :            (au)                 -4.429348802668
 X-C energy              :            (au)                -13.684784979990
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  6.137562738273
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -212.346203307903
 Nuclear energy          :            (au)                  6.137562738273
---------------------------------------------------------------------------
 Total energy            :            (au)             -2.062086405696e+02
                         :      (kcal/mol)             -1.293978745471e+05
                         :        (kJ/mol)             -5.414007074563e+05
                         :            (eV)             -5.611222719177e+03
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -46.575036019659
    1    1    a          :            (au)                 -3.096365293139
    2    1    a          :            (au)                 -2.011936886419
    3    1    a          :            (au)                 -1.817727609234
    4    1    a          :            (au)                 -1.817726234139
    5    1    a          :            (au)                 -1.816465637167
    6    1    a          :            (au)                 -0.200487599294
    7    1    a          :            (au)                 -0.119492142394
    8    1    b          :            (au)                -46.574087610188
    9    1    b          :            (au)                 -3.094473182322
   10    1    b          :            (au)                 -2.003954751981
   11    1    b          :            (au)                 -1.816631618050
   12    1    b          :            (au)                 -1.816630346371
   13    1    b          :            (au)                 -1.813363355805
   14    1    b          :            (au)                 -0.183654495064
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -114.758032781227
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          3.872983e+00        -206.208640569630         -2.062086e+02
    1          1.844127e-01        -207.399922779615         -1.191282e+00
    2          5.651879e-02        -207.438068973205         -3.814619e-02
    3          4.633728e-02        -207.443468086883         -5.399114e-03
    4          3.194548e-02        -207.445812854354         -2.344767e-03
    5          5.403608e-02        -207.447145849596         -1.332995e-03
    6          1.897996e-02        -207.447530674578         -3.848250e-04
    7          2.031373e-02        -207.447984882343         -4.542078e-04
    8          8.937260e-03        -207.448052238716         -6.735637e-05
    9          8.440120e-03        -207.448097816653         -4.557794e-05
   10          5.873372e-03        -207.448118850233         -2.103358e-05
   11          4.351820e-03        -207.448131248341         -1.239811e-05
   12          1.924409e-03        -207.448136028788         -4.780448e-06
   13          1.282661e-03        -207.448137386152         -1.357363e-06
   14          1.173358e-03        -207.448138140473         -7.543209e-07
   15          1.434600e-03        -207.448139137412         -9.969394e-07
   16          8.997843e-04        -207.448139624137         -4.867246e-07
   17          8.008269e-04        -207.448139876386         -2.522489e-07
   18          7.698537e-04        -207.448140106580         -2.301946e-07
   19          6.819777e-04        -207.448140259107         -1.525266e-07
   20          5.445603e-04        -207.448140323606         -6.449932e-08
   21          2.361822e-04        -207.448140355241         -3.163464e-08
   22          1.668322e-04        -207.448140380408         -2.516717e-08
   23          1.378760e-04        -207.448140395857         -1.544916e-08
   24          8.597415e-05        -207.448140400606         -4.748898e-09
---------------------------------------------------------------------------
                      SCF converged in 24 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Li            :        0.000000        0.000000       -4.562551
    1      Mg            :        0.000000        0.000000        1.302969
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                207.033116048927
 E-N energy              :            (au)               -508.517123225403
 Coulomb energy          :            (au)                106.061685234833
 Exchange energy         :            (au)                 -4.438599313877
 X-C energy              :            (au)                -13.724781883359
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  6.137562738273
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -213.585703138879
 Nuclear energy          :            (au)                  6.137562738273
---------------------------------------------------------------------------
 Total energy            :            (au)             -2.074481404006e+02
                         :      (kcal/mol)             -1.301756724278e+05
                         :        (kJ/mol)             -5.446550137915e+05
                         :            (eV)             -5.644951226347e+03
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -46.910875224990
    1    1    a          :            (au)                 -3.144342337417
    2    1    a          :            (au)                 -2.042182482439
    3    1    a          :            (au)                 -1.866208775341
    4    1    a          :            (au)                 -1.866208769308
    5    1    a          :            (au)                 -1.865439010088
    6    1    a          :            (au)                 -0.218064220050
    7    1    a          :            (au)                 -0.130760312384
    8    1    b          :            (au)                -46.910294961329
    9    1    b          :            (au)                 -3.143334666678
   10    1    b          :            (au)                 -2.034021007216
   11    1    b          :            (au)                 -1.865642729877
   12    1    b          :            (au)                 -1.865642694005
   13    1    b          :            (au)                 -1.863630632469
   14    1    b          :            (au)                 -0.205853445226
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -115.932501268818
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000000       -0.000000       -2.297647
 Magnitude               :            (au)                        2.297647
                         :         (Debye)                        5.840036
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        1.947978
 Magnitude               :            (au)                        1.947978
                         :         (Debye)                        4.951265
---------------------------------------------------------------------------
 Total vector            :        0.000000       -0.000000       -0.349670
 Magnitude               :            (au)                        0.349670
                         :         (Debye)                        0.888772
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 24m 50s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
